libgalois/src/Version.cpp.in

```diff
@@ -22,14 +22,14 @@
 #define QUOTE(name) #name
 #define STR(macro) QUOTE(macro)
 
```

Comment:
Not necessarily needed in this PR, but we should have a similar set of macros for detecting the version of the Galois headers used at compile time. These functions just tell us the version of the library used.